tutorials/vscodeJavaSetup.md

```diff
@@ -0,0 +1,182 @@
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+This tutorial will guide you through creating and managing Java projects in VS Code.
+
+<box type="info" seamless>
+
+This tutorial is adapted from the [official VS Code Java Tutorial](https://code.visualstudio.com/docs/java/java-tutorial).
+</box>
+
+## Install the Extension Pack for Java
+
+To develop Java applications in VS Code, you need to install the appropriate extensions that provide language support, debugging, and other essential features. The easiest way to get started with Java development in VS Code is to install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes all the essential extensions you need.
+
+### Installation steps
+
+1. **Open the Extensions View**: {{ icon_windows }}/{{ icon_linux}} `Ctrl+Shift+X` | {{ icon_apple }} `Cmd+Shift+X`
+2. **Search for** `Extension Pack for Java`
+3. **Click** `Install` on the extension by Microsoft
+
+### What's Included
+
+The [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack) includes these essential tools for Java development:
+* **Language Support for Java** - Syntax highlighting, IntelliSense, and error detection
+* **Debugger for Java** - Set breakpoints and step through your code
+* **Test Runner for Java** - Run and debug JUnit tests
+* **Maven for Java** - Support for Maven projects
+* **Gradle for Java** - Support for Gradle projects
+* **Project Manager for Java** - Manage Java projects and dependencies
+* **IntelliCode** - AI-assisted code completion
+
+<box type="tip" seamless>
+
+**Quick tip:** After installation, open the Command Palette: {{ icon_windows }}/{{ icon_linux}} `Ctrl+Shift+P` | {{ icon_apple }} `Cmd+Shift+P`, type `Java: Tips for Beginners` and open it for helpful information to get started.
+</box>
+
+### Verify installation
+
+After installing the extensions, you should see:
+
+1. **Java language support** - `.java` files will have syntax highlighting
+2. **Java commands** in the Command Palette - Open the Command Palette: {{ icon_windows }}/{{ icon_linux}} `Ctrl+Shift+P` | {{ icon_apple }} `Cmd+Shift+P` and type `Java:` to see available commands
+3. **Status bar indicators** - Java information in the bottom status bar
+
+## Setting up Java Development Kit (JDK)
+
+Before you can develop Java applications in VS Code, you need to have a Java Development Kit (JDK) installed and properly configured.
+
+### Install JDK
+
+VS Code can help you install and configure a JDK automatically through its integrated tools.
+
+1. **Open the Command Palette**: {{ icon_windows }}/{{ icon_linux}} `Ctrl+Shift+P` | {{ icon_apple }} `Cmd+Shift+P`
+2. **Type** `Java: Install New JDK` and select it
+3. **Choose a JDK version** (Java 17 or later recommended)
+4. **Follow the installation prompts**
+5. **Wait for the download and installation** to complete
```

Comment:
I'd rather they install the JDK outside of VSC, to learn JDK installation independently from the IDE, and to ensure the precise version we want is installed.